CHANGES.md

```diff
@@ -1,6 +1,10 @@
 Change Log
 ==========
 
+### v7.6.4
+
+* Fix a bug where `ArcGisMapServerCatalogItem` legends were being created during search.
```

Comment:
Just gonna shift this to 7.6.3 as we're still on 7.6.2!

Reply:
No worries. I always get confused what number I put stuff under :)